src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -62,6 +67,30 @@ public double getLastLastCenterLimelightY() {
     return lastLastCenterLimelightY;
   }
 
+  @AutoLogOutput
```

Comment:
explanation of the math would be helpful 👍

Reply:
I'll write a design doc on here rq